.github/workflows/publish.yaml

```diff
@@ -17,7 +17,7 @@ jobs:
     - name: Install dependencies
       run: |
         sudo snap install charm --classic
-        sudo pip3 install charmcraft
+        sudo pip3 install charmcraft==latest
```

Comment:
By default pip installs the latest version, isn't? You can either pick a version or not have this (I don't think `==latest` works)

Reply:
I'd recommend using the snap with `--channel=latest/candidate`. The pip package is not intended to actually be used.